var/spack/repos/builtin/packages/libuv-julia/package.py

```diff
@@ -9,15 +9,13 @@ class LibuvJulia(AutotoolsPackage):
     """Multi-platform library with a focus on asynchronous IO"""
 
     homepage = "https://libuv.org"
-    url = "https://github.com/JuliaLang/libuv/archive/refs/heads/julia-uv2-1.44.1.tar.gz"
+    url = "https://github.com/JuliaLang/libuv/archive/refs/heads/julia-uv2-1.44.2.tar.gz"
+    git = "https://github.com/JuliaLang/libuv.git"
 
-    # julia's libuv fork doesn't tag releases, only has release branches, so we
-    # fix commits.
-    version(
-        "1.44.1",
-        sha256="f931e7825702cbb6d07486d92e5436990cf20f91e2b56d6f759822c0f832b13e",
-        url="https://github.com/JuliaLang/libuv/archive/1b2d16477fe1142adea952168d828a066e03ee4c.tar.gz",
-    )
+    # julia's libuv fork doesn't tag (all?) releases, so we fix commits.
+    version("1.44.2", commit="e6f0e4900e195c8352f821abe2b3cffc3089547b")
```

Comment:
This doesn't match what is tagged as the stable commit for this release (https://github.com/JuliaLang/libuv/releases/tag/v1.44.2).

```suggestion
    version("1.44.2", commit="0c1fa696aa502eb749c2c4735005f41ba00a27b8")
```